crates/tests-e2e/tests/restore_from_backup.rs

```diff
@@ -0,0 +1,51 @@
+use anyhow::Result;
+use native::api;
+use tests_e2e::app::run_app;
+use tests_e2e::logger::init_tracing;
+use tests_e2e::setup;
+use tokio::task::spawn_blocking;
+
+#[tokio::test]
+#[ignore = "need to be run with 'just e2e' command"]
+async fn app_can_be_restored_from_a_backup() -> Result<()> {
```

Comment:
🙃 I think the test should go with the feature so that one can check out the patch that adds the feature knowing that it was meant to work.